src/Invariants/Invariants.jl

```diff
@@ -23,7 +23,7 @@ The module allows for the computation of complex graph invariants and operations
 ```julia
 julia> using GraphProperties.Invariants
 
-julia> ind_number = compute(IndependenceNumber, my_graph)
+julia> α = compute(IndependenceNumber, my_graph)
```

Comment:
This is kind of silly, but a lot of people use \alpha for the independence number of a graph.